src/integrationTest/java/uk/gov/hmcts/reform/notificationservice/data/NotificationRepositoryTest.java

```diff
@@ -51,12 +46,48 @@ void should_save_and_read_notification() {
                 assertThat(n.errorDescription).isEqualTo(newNotification.errorDescription);
                 assertThat(n.createdAt).isNotNull();
                 assertThat(n.processedAt).isNull();
-                assertThat(n.status).isEqualTo(NotificationStatus.PENDING);
+                assertThat(n.status).isEqualTo(PENDING);
             });
     }
 
     @Test
     void should_return_empty_optional_when_there_is_no_notification_in_db() {
         assertThat(notificationRepository.find(1_000)).isEmpty();
     }
+
+    @Test
+    void should_all_pending_notifications_to_be_sent_out() {
```

Comment:
There should also be a test for multiple notifications verifying also that notifications with another status are not included in the result.

Reply:
It should. At the moment it is not a part of the path of development nor ticket on it's own. Can be extended in future to verify such case - no doubt.

P.S. I see I didn't update the method name - will do now